graphene/src/layers/style/mod.rs

```diff
@@ -83,16 +108,26 @@ impl PathStyle {
 	pub fn clear_stroke(&mut self) {
 		self.stroke = None;
 	}
+	pub fn view_mode(&mut self, new_mode: ViewMode) {
+		self.view_mode = new_mode;
+	}
+	pub fn get_view_mode(&self) -> ViewMode {
+		self.view_mode
+	}
+
```

Comment:
The rust naming convention discourages explicit `get` functions in most cases.
These should probably be `view_mode` and `update_view_mode` to stay consistent with the naming on Document